packages/vats/src/lib-chainStorage.js

```diff
@@ -36,7 +36,7 @@ harden(sanitizePathSegment);
  * Create a root storage node for a given backing function and root path.
  *
  * @param {(message: StorageMessage) => any} handleStorageMessage a function for sending a storageMessage object to the storage implementation (cf. golang/cosmos/x/vstorage/vstorage.go)
```

Comment:
Given this relationship with vstorage.go, I'd prefer to keep the ChainStorageNode type and drop StorageNode rather than vice versa.